Build fails on MacOS

I'm unable to build proxy master (59ee751a68f26d5218adf2a12df0ccee9f1fa404) on MacOS High Sierra (10.10.13). Output:

```
ERROR: /Users/theganyo/dev/istio/src/istio.io/proxy/src/envoy/mixer/BUILD:23:1: C++ compilation of rule '//src/envoy/mixer:filter_lib' failed (Exit 1)
src/envoy/mixer/tcp_filter.cc:187:8: error: 'GetSourceIpPort' overrides a member function but is not marked 'override' [-Werror,-Winconsistent-missing-override]
  bool GetSourceIpPort(std::string* str_ip, int* port) const {
       ^
external/mixerclient_git/control/include/tcp/check_data.h:32:16: note: overridden virtual function is here
  virtual bool GetSourceIpPort(std::string* ip, int* port) const = 0;
               ^
src/envoy/mixer/tcp_filter.cc:191:8: error: 'GetSourceUser' overrides a member function but is not marked 'override' [-Werror,-Winconsistent-missing-override]
  bool GetSourceUser(std::string* user) const {
       ^
external/mixerclient_git/control/include/tcp/check_data.h:35:16: note: overridden virtual function is here
  virtual bool GetSourceUser(std::string* user) const = 0;
               ^
src/envoy/mixer/tcp_filter.cc:194:8: error: 'GetDestinationIpPort' overrides a member function but is not marked 'override' [-Werror,-Winconsistent-missing-override]
  bool GetDestinationIpPort(std::string* str_ip, int* port) const {
       ^
external/mixerclient_git/control/include/tcp/report_data.h:33:16: note: overridden virtual function is here
  virtual bool GetDestinationIpPort(std::string* ip, int* port) const = 0;
               ^
src/envoy/mixer/tcp_filter.cc:202:8: error: 'GetReportInfo' overrides a member function but is not marked 'override' [-Werror,-Winconsistent-missing-override]
  void GetReportInfo(
       ^
external/mixerclient_git/control/include/tcp/report_data.h:41:16: note: overridden virtual function is here
  virtual void GetReportInfo(ReportInfo* info) const = 0;
               ^
4 errors generated.
INFO: Elapsed time: 228.389s, Critical Path: 25.66s
FAILED: Build did NOT complete successfully
make: *** [build] Error 1
```

--- a/src/envoy/mixer/tcp_filter.cc
+++ b/src/envoy/mixer/tcp_filter.cc
@@ -199,8 +199,8 @@ class TcpInstance : public Network::Filter,
     }
     return false;
   }
-  void GetReportInfo(
-      ::istio::mixer_control::tcp::ReportData::ReportInfo* data) const override {
+  void GetReportInfo(::istio::mixer_control::tcp::ReportData::ReportInfo* data)
+      const override {
     data->received_bytes = received_bytes_;
     data->send_bytes = send_bytes_;
     data->duration = std::chrono::duration_cast<std::chrono::nanoseconds>(

--- a/src/envoy/mixer/tcp_filter.cc
+++ b/src/envoy/mixer/tcp_filter.cc
@@ -184,14 +184,14 @@ class TcpInstance : public Network::Filter,
   void onAboveWriteBufferHighWatermark() override {}
   void onBelowWriteBufferLowWatermark() override {}
 
-  bool GetSourceIpPort(std::string* str_ip, int* port) const {
+  bool GetSourceIpPort(std::string* str_ip, int* port) const override {
     return Utils::GetIpPort(
         filter_callbacks_->connection().remoteAddress()->ip(), str_ip, port);
   }
-  bool GetSourceUser(std::string* user) const {
+  bool GetSourceUser(std::string* user) const override {
     return Utils::GetSourceUser(&filter_callbacks_->connection(), user);
   }
-  bool GetDestinationIpPort(std::string* str_ip, int* port) const {
+  bool GetDestinationIpPort(std::string* str_ip, int* port) const override {
     if (filter_callbacks_->upstreamHost() &&
         filter_callbacks_->upstreamHost()->address()) {
       return Utils::GetIpPort(
@@ -200,7 +200,7 @@ class TcpInstance : public Network::Filter,
     return false;
   }
   void GetReportInfo(
-      ::istio::mixer_control::tcp::ReportData::ReportInfo* data) const {
+      ::istio::mixer_control::tcp::ReportData::ReportInfo* data) const override {
     data->received_bytes = received_bytes_;
     data->send_bytes = send_bytes_;
     data->duration = std::chrono::duration_cast<std::chrono::nanoseconds>(

